sdk/textanalytics/Azure.AI.TextAnalytics/api/Azure.AI.TextAnalytics.netstandard2.0.cs

```diff
@@ -479,6 +662,38 @@ public enum TextSentiment
         Mixed = 3,
     }
 }
+namespace Azure.AI.TextAnalytics.Models
```

Comment:
it will be better to move all types to main namespace and later, after we do the design start moving things around